synapse/storage/data_stores/main/event_push_actions.py

```diff
@@ -37,6 +39,18 @@
 ]
 
 
+@attr.s
+class EventPushSummary(object):
```

Comment:
this is the default as of python 3, best to omit it for consitency.

```suggestion
class EventPushSummary():
```